sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnectionAsync.java

```diff
@@ -0,0 +1,442 @@
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Add a participant to the call.
+     *
+     * @param participant Invited participant.
+     * @param alternateCallerId The phone number to use when adding a phone number participant.
+     * @param operationContext The value to identify context of the operation. This is used to co-relate other
+     *                         communications related to this operation
+     * @return response for a successful add participant request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> addParticipantWithResponse(
+        CommunicationIdentifier participant,
+        String alternateCallerId,
+        String operationContext) {
+        return addParticipantWithResponse(participant, alternateCallerId, operationContext, Context.NONE);
+    }
+
+    Mono<Response<Void>> addParticipantWithResponse(
+        CommunicationIdentifier participant,
+        String alternateCallerId,
+        String operationContext,
+        Context context) {
+        try {
+            Objects.requireNonNull(participant, "'participant' cannot be null.");
+            InviteParticipantsRequest request =
+                InviteParticipantRequestConverter
+                    .convert(participant, alternateCallerId, operationContext, null);
+            return withContext(contextValue -> {
+                contextValue = context == null ? contextValue : context;
+                return callConnectionInternal
+                    .inviteParticipantsWithResponseAsync(callConnectionId, request, contextValue)
+                    .onErrorMap(CommunicationErrorException.class, CallingServerErrorConverter::translateException);
+            });
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Remove a participant from the call.
+     *
+     * @param participantId Participant id.
+     * @return response for a successful remove participant request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> removeParticipant(String participantId) {
+        try {
+            Objects.requireNonNull(participantId, "'participantId' cannot be null.");
+            return callConnectionInternal.removeParticipantAsync(callConnectionId, participantId)
+                .onErrorMap(CommunicationErrorException.class, CallingServerErrorConverter::translateException);
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Remove a participant from the call.
+     *
+     * @param participantId Participant id.
+     * @return response for a successful remove participant request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> removeParticipantWithResponse(String participantId) {
+        return removeParticipantWithResponse(participantId, Context.NONE);
```

Comment:
If you put in Context.NONE here, then users will not be able to pass in their custom Context value. It should be null instead, so that we can use withContext (line 432)

Reply:
null is put as context value here and other places